packages/components/addon/components/hds/side-nav/index.js

```diff
@@ -11,7 +11,8 @@ import { registerDestructor } from '@ember/destroyable';
 
 export default class HdsSideNavComponent extends Component {
   @tracked isResponsive = this.args.isResponsive ?? true;
-  @tracked isMinimized = this.isResponsive; // we set it minimized by default so that if we switch viewport from desktop to mobile its already minimized
+  @tracked isMinimized = this.args.isMinimized ?? false;
```

Comment:
`isMinimized` used to be an internal variable with meaning only on mobile screens (when `isDesktop` was `false` and `isResponsive was `true`). it was, indeed, set to `true` by default so that if the application starts on a mobile screen the SideNav will be minimized. Now that the collapsible feature may be available on any screen size, we adjusted the meaning of this variable to reflect the minimized state independent of the screen size and made it part of the public API so applications can programmatically set the default state (for state persistence purposes). The behavior on mobile screens is preserved because we do a media query check as part of the constructor of this component.

Reply:
Makes sense. Can you maybe add a short comment to make sure this information is preserved (for future reference)? Thanks

Reply:
Yes, added some clarity around arguments in ae32fba6a70761bb7ad220b29e1db897b208b98e